src/spoon/nodes.clj

```diff
@@ -7,5 +7,20 @@
 (defn get-node [org node & [options]]
   (client/api-request :get "/organizations/%s/nodes/%s" [org node] options))
 
+(defn update-node
+  [org node & [options]]
+  (let [data (assoc (get-node org node options) :data (:data options))]
+    (client/api-request :put "/organizations/%s/nodes/%s" [org node] (assoc options {:data data}))))
```

Comment:
I think you just want `data` here as opposed to
```clojure
(assoc options {:data data})
```
since you already added it into the map on the previous line. Have you tried these out at the repl yet?